### DISCUSSION TITLE
Qualified for student approved in Oct 11, but still not getting my access to benefits

### DISCUSSION THREAD

**Discussion by FafaPicker** [community/community#141856] [GitHub Education]
### Hi everyone,

![20241018-112515](https://github.com/user-attachments/assets/73a3cd3d-2cb3-4fc6-891f-3f412e0ef4b6)
Do I need to upload more certificates or I should just wait?
Any helps will be grateful

**Comment by davevad93** [comment]
Hi @MushroomInTheTop ,

Don't worry, you just have to wait. When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @MushroomInTheTop was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
